### PR TITLE
Student Educational Homework API

### PR DESCRIPTION
**Pull request by osagiestar**
Student edu_homework API completed

### CODE REVIEW

**Review by ISTANBULBEKLE — APPROVED** [review]
Great Job, thanks

File: src/routes/students/routes.js [hunk 1]
```diff
@@ -12,5 +13,6 @@ const router = express.Router();
 router.get("/tracker", studentTracker);
 router.get("/profile/:profile_id", getStudentProfile);
 router.post("/pd_skills", postPdSkills);
+router.post("/student/edu_homework", postStudentEduHomework);
```
Comment (sadatakhtar) [inline src/routes/students/routes.js]: End point route needs to be /edu_homework instead of /students/edu_homework.

**Review by sadatakhtar — CHANGES_REQUESTED** [review]
Please change the end point to /edu_homework

**Review by sadatakhtar — APPROVED** [review]

**Review by sadatakhtar — APPROVED** [review]

**Comment by ISTANBULBEKLE** [comment]
Guys, when there is 2 approval please just continue with Merging.

**Pull request merged**